### ISSUE TITLE
R-Ladies Smithsonian?

### ISSUE THREAD

**Issue by maurolepore** [si-library/meetups#3]
@bullardk, 

One other thing you may want to know about is the [mission of R-Ladies](https://rladies.org/about-us/help/).

See what you think and let me know if you think the library could support the mission in any way -- maybe by providing the space for Smithsonian R-Ladies to meet. I know several of the most influential R-Ladies around the world. I'm sure they could inspire and help many woman at SI.

**Comment by bullardk** [comment]
@maurolepore 

This sounds like an excellent idea! I'd like to talk to the SI R-Ladies about ways that the Libraries can support their community and their mission.

**Comment by maurolepore** [comment]
Great! There is no SI R-Ladies yet, but I have discussed it with a number of women from several SI facilities and they seem interested. I'll see who is willing to take the lead and will send her to you.

There is great potential for R-Ladies Smithsonian to take off.  I think the library could catalyze the process by reminding them how great of a resource you are. The biggest challenge I hear R-Ladies face is finding a space and spreading the word. And that is what you, @bullardk, are already doing. There is nothing to lose and a lot to gain 👍

**Comment by bullardk** [comment]
@maurolepore 

I totally agree! I look forward to connecting with the SI R-Lady that you send my way.

**Comment by maurolepore** [comment]
@gonzalezeb, @hmullerlandau, @jess-shue, @JessicaNeedham, @kccushman, @mopicon, @ngokangmin, @overstreeth, @RutujaCT, @teixeirak, @ValentineHerr, @lucymchang, @megbalk,

You may be interested in this thread if this applies to you:

* You use or would like to learn R.
* You identify yourself as a woman.
* You work closely with the Smithsonian.

> R-LADIES IS A WORLD-WIDE ORGANIZATION TO PROMOTE GENDER DIVERSITY IN THE R COMMUNITY...If you are an R-Lady and want to start a chapter ... please get in touch via info@rladies.org

-- https://rladies.org/about-us/help

There is no R-Ladies Smithsonian, and I think you could start it. 



Three challenges in starting/maintaining a chapter are finding a good space to meet, allowing remote participation, and advertising the events. The great news is this: All of that you can get through the Smithsonian Libraries.

If you or someone you know may be interested in starting R-Ladies Smithsonian, I strongly recommend you contact Kristen Bullard (referring to her as @bullardk) to find out how the library can support you.

https://github.com/rladies/starter-kit

**Comment by maurolepore** [comment]
I've just learned from @lauracion that chapters of RLadies must be associated to cities not institutions. Sorry for steering you in the wrong direction.

Instead, you could engage with RLadies in your own city:

* in DC: https://twitter.com/RLadiesDC?lang=en
* in NYC https://twitter.com/RLadiesNYC?lang=en
* No RLadies yet in Panama city, Panama.


@bullardk, would it be possible to invite to the library an R Lady speaker who is not a Smithsonian employee? (If not, we could invite her remotely).

**Comment by bullardk** [comment]
@maurolepore and @lauracion, yes, we regularly host outside speakers. I'd be happy to work with you on something like that.
